openshift_scalability/config/golang/pyconfigMasterVertScale.yaml

```diff
@@ -4,6 +4,7 @@ ClusterLoader:
   projects:
     - num: 5
       basename: clusterproject
+      ifexists: delete
```

Comment:
For master vertical scale test, we don't delete the projects until the end of all iterations. Instead of deleting the projects created in the first iteration, we add more projects to match the total count in order to avoid duplication and  long test duration.